gdal/frmts/rmf/rmfdataset.cpp

```diff
@@ -2891,7 +2911,8 @@ CPLErr RMFDataset::WriteRawTile(int nBlockXOff, int nBlockYOff,
 
 CPLErr RMFDataset::ReadTile(int nBlockXOff, int nBlockYOff,
                             GByte* pabyData, size_t nRawBytes,
-                            GUInt32 nRawXSize, GUInt32 nRawYSize)
+                            GUInt32 nRawXSize, GUInt32 nRawYSize, 
+                            bool& bNullTile)
```

Comment:
it would be safer to set bNullTile = false at the beginning of the method (if the caller didn't initialize the variable)

Reply:
Yes, it will be clearer for future generations.